cobalt/build/gn.py

```diff
@@ -96,19 +96,22 @@ def write_build_args(build_args_path, original_lines, dict_settings,
 
 
 def main(out_directory: str, platform: str, build_type: str,
-         overwrite_args: bool, gn_gen_args: List[str]):
+         gn_gen_args: List[str]):
   platform_path = f'cobalt/build/configs/{platform}'
   dst_args_gn_file = os.path.join(out_directory, 'args.gn')
   src_args_gn_file = os.path.join(platform_path, 'args.gn')
   Path(out_directory).mkdir(parents=True, exist_ok=True)
 
-  if overwrite_args or not os.path.exists(dst_args_gn_file):
-    build_args = get_build_args(src_args_gn_file)
-    write_build_args(dst_args_gn_file, build_args[0], build_args[1], build_type)
-  else:
-    print(f'{dst_args_gn_file} already exists.' +
-          ' Running ninja will regenerate build files automatically.')
-
+  build_args = get_build_args(src_args_gn_file)
+  if os.path.exists(dst_args_gn_file):
+    # Copy the stale args.gn into stale_args.gn
+    stale_dst_args_gn_file = dst_args_gn_file.replace('args', 'stale_args')
+    os.rename(dst_args_gn_file, stale_dst_args_gn_file)
+    print(f' Warning: {dst_args_gn_file} is rewritten.' +
```

Comment:
nit: can't we just use implicit string concatenation here?

Reply:
ahh nice , this actually caught a bug, im now usiong implicit string concatenation.
the second string was not a fstring so {stale_dst_args_gn_file} was getting printed out as is.